docs/content/users/install/ddev-installation.md

```diff
@@ -219,10 +228,10 @@ Once you’ve [installed a Docker provider](docker-installation.md), you’re re
 
     If you prefer to install DDEV manually on WSL2, it's not hard. These techniques can be used to install on a non-Ubuntu distro, and can be adapted for Yum-based or ArchLinux-based distros as well:
 
-    1. In an administrative PowerShell, run `mkcert -install` and follow the prompt to install the Certificate Authority.
+    1. In PowerShell, run `mkcert -install` and follow the prompt to install the Certificate Authority.
     2. In an administrative PowerShell, run `$env:CAROOT="$(mkcert -CAROOT)"; setx CAROOT $env:CAROOT; If ($Env:WSLENV -notlike "*CAROOT/up:*") { $env:WSLENV="CAROOT/up:$env:WSLENV"; setx WSLENV $Env:WSLENV }`. This will set WSL2 to use the Certificate Authority installed on the Windows side. In some cases it takes a reboot to work correctly.
-    3. In administrative PowerShell, run `wsl --install <distro>`, for example `wsl --install Debian`. This will install the WSL2 distro for you.
-    5. **Docker CE:** Follow the instructions in the [Linux install section here](docker-installation.md#linux) to install Docker CE and DDEV.
+    3. In administrative PowerShell, run `wsl --install <distro> --name DDEV`, for example `wsl --install Debian`. This will install the WSL2 distro for you. (The name "DDEV" is just a suggestion; you can use any name you like.)
```

Comment:
```suggestion
    3. In administrative PowerShell, run `wsl --install <distro> --name DDEV`, for example `wsl --install Debian --name DDEV`. This will install the WSL2 distro for you. (The name "DDEV" is just a suggestion; you can use any name you like.)
```